src/locale/de.json

```diff
@@ -262,4 +262,5 @@
     "Import in a new layer": "In eine neue Ebene importieren",
     "Layer": "Ebene",
     "Please choose a format": "Bitte wähle ein Format"
```

Comment:
Note that those file are automatically generated using Leaflet.i18n. You should have a "make i18n" command in the Makefile.